src/alt.js

```diff
@@ -454,6 +461,7 @@ function createStoreFromClass(alt, StoreModel, key, ...argsForConstructor) {
 
 class Alt {
   constructor(config = {}) {
+    this.config = config
```

Comment:
I think it makes more sense to store `this.stateKey = config.stateKey` rather than the entire config.